pyanaconda/modules/timezone/timezone.py

```diff
@@ -17,13 +17,15 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
+from pyanaconda.core.configuration.anaconda import conf
 from pyanaconda.dbus import DBus
 from pyanaconda.core.signal import Signal
 from pyanaconda.modules.common.base import KickstartModule
 from pyanaconda.modules.common.constants.services import TIMEZONE
 from pyanaconda.modules.common.containers import TaskContainer
 from pyanaconda.modules.timezone.timezone_interface import TimezoneInterface
 from pyanaconda.modules.timezone.kickstart import TimezoneKickstartSpecification
+from pyanaconda.modules.timezone.installation import ConfigureTimezoneTask
```

Comment:
Please try to sort the imports alphabetically. It doesn't have to be perfect but it is better to read, so reasonable effort is enough :).